internal/db/db.go

```diff
@@ -133,6 +136,10 @@ const (
 			WHERE composes.org_id=$2)`
 )
 
+func newComposeEntry(jobId uuid.UUID, blueprintId uuid.UUID, version int, request json.RawMessage, createdAt time.Time, imageName *string, clientId *string) ComposeEntry {
+	return ComposeEntry{Id: jobId, BlueprintId: blueprintId, BlueprintVersion: version, Request: request, CreatedAt: createdAt, ImageName: imageName, ClientId: clientId}
+}
```

Comment:
Alert, alert. This is ugly. Why would you do this? :-D

Just use struct literal syntax, this is very anti Go idiomatic.